common/dependabot-common.gemspec

```diff
@@ -10,7 +10,7 @@ Gem::Specification.new do |spec|
                       "Python, PHP, Elixir, Rust, Java, .NET, Elm and Go"
 
   spec.author       = "Dependabot"
-  spec.email        = "support@dependabot.com"
+  spec.email        = "opensource@github.com"
```

Comment:
it seems we have a lot of spec files that need to be touched?

Reply:
Thanks for the thorough review to doublecheck for other usages! ✨ 

Spec files are test files, so it's okay to leave them as-is.

I did the same search, and the only one I saw that isn't spec/fixtures is:
* https://github.com/dependabot/dependabot-core/issues/6968

But that's out of scope for this particular PR, so IMO we should merge this one as-is and handle that as a separate issue.

Reply:
Also, feel free to correct me if you see other usages that aren't tests / fixtures.